exe/puppet-query.cc

```diff
@@ -35,6 +35,9 @@ main(int argc, char **argv) {
         po::options_description global_options("global options");
         global_options.add_options()
                 ("help,h", "produce help message")
+                ("pretty,p",
+                 po::value<bool>()->default_value(true),
```

Comment:
Fwiw I think if we did have this option we would just have a flag with no value called `--no-pretty`  or `--ugly` :) and the default would automatically be false (which corresponds to true for the `--pretty` default).